specification/iothub/resource-manager/readme.md

```diff
@@ -132,9 +141,23 @@ output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-iothub
 
 ``` yaml $(java) && $(multiapi)
 batch:
+  - tag: package-2018-12
   - tag: package-2018-04
 ```
 
+### Tag: package-2018-12 and java
+
+These settings apply only when `--tag=package-2018-12 --java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(tag) == 'package-2018-12' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.iothub.v2018_12_01
+  output-folder: $(azure-libraries-for-java-folder)/iothub/resource-manager/v2018_12_01

```

Comment:
must have preview as suffix like this: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/consumption/resource-manager#tag-package-2017-04-preview-and-java

Reply:
same for namespace. I would recommend having -preview on the tag as well.

Reply:
Fixed